src/Controller/CRUDController.php

```diff
@@ -1534,6 +1534,11 @@ protected function preList(Request $request)
         return $this->get('translator')->trans($id, $parameters, $domain, $locale);
     }
 
+    private function getSelectedTab(Request $request)
+    {
+        return array_filter(['_tab' => $request->request->get('_tab')]);
```

Comment:
don't you still need to call `has`? won't it throw an execption

Reply:
No it won't: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/ParameterBag.php#L84